SidecarSubtitles/BasicSidecarSubtitlesPlayer/objc/BasicSidecarSubtitlesPlayer/ViewController.m

```diff
@@ -144,12 +144,33 @@ - (NSArray *)setupSubtitles
     NSArray *textTracks =
     @[
       @{
+          // required tracks descriptor: kBCOVSSTextTracksKindSubtitles or kBCOVSSTextTracksKindCaptions
           kBCOVSSTextTracksKeyKind: kBCOVSSTextTracksKindSubtitles,
-          kBCOVSSTextTracksKeyLabel: @"English",
-          kBCOVSSTextTracksKeyDefault: @YES,
+          
+          // required language code
           kBCOVSSTextTracksKeySourceLanguage: @"en",
+          
+          // required display name
+          kBCOVSSTextTracksKeyLabel: @"English",
+          
+          // required: source URL of WebVTT file or playlist as NSString
           kBCOVSSTextTracksKeySource: @"http://players.brightcove.net/3636334163001/ios_native_player_sdk/vtt/sample.vtt",
-          kBCOVSSTextTracksKeyMIMEType: @"text/vtt"
+          
+          // optional MIME type
+          kBCOVSSTextTracksKeyMIMEType: @"text/vtt",
+          
+          // optional "default" indicator
+          kBCOVSSTextTracksKeyDefault: @YES,
+          
+          // duration is required for WebVTT URLs (ending in ".vtt");
+          // optional for WebVTT playlists (ending in ".m3u8")
```

Comment:
Dots or no dots. Up to you. 😄 PS: I was not on the list of reviewers for this one though I eventually saw the @me in slack.

Reply:
.√